components/reset_status_modal/reset_status_modal.tsx

```diff
@@ -77,11 +111,11 @@ export default class ResetStatusModal extends React.PureComponent {
         );
     }
 
-    hideModal = () => {
+    private hideModal = (): void => {
```

Comment:
this can be immediately returned
```suggestion
    private hideModal = (): void => this.setState({show: false});
```